.circleci/config.yml

```diff
@@ -113,11 +107,18 @@ jobs:
             - run:
                 name: Run UI tests against earlier supported major IntelliJ versions
```

Comment:
Pls unlock this step for the sake of tests so that we can see the effect when multiple versions are executed (and only one of them &mdash; not the first one) fails

Reply:
it is a pity that it is not packed in directories for easier viewing

Reply:
I run only two versions but with e.g. 4 or 5 it will be a lot of pictures. Maybe better to keep pictures only from one version ? (by cleaning `/tmp/ide-probe/screenshots/` at the beginning of each testSuite?)

Reply:
or no ;p Firstly I will check tomorrow if it is any possibility to store folders on circleCI

Reply:
https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/5184/workflows/238460c5-e5b7-463b-9e88-a30ea48f45fe/jobs/5445/artifacts 
I did something like this

Reply:
gr8! looks good

Reply:
but this zip contains directory /tmp/ide-probe/screenshots ... maybe I will try to flatten this structure

Reply:
Oh! would be nice, yeah